spec/requests/api/providers_spec.rb

```diff
@@ -1116,4 +1116,14 @@ def gen_import_request
       end
     end
   end
+
+  context "#options" do
+    it "returns options for queried provider class" do
+      api_basic_authorize
+      ExtManagementSystem.descendants.select { |ems| ems.respond_to?(:provider_settings) }.each do |ems_type|
```

Comment:
one last thing 😄  you can simplify the spec by using `expect_options_results`, example here: https://github.com/ManageIQ/manageiq-api/blob/master/spec/requests/api/authentications_spec.rb#L480